textproc/pandoc/Portfile

```diff
@@ -24,33 +25,36 @@ long_description    \
     Textile, groff man pages, plain text, Emacs Org-Mode, AsciiDoc, EPUB (v2 \
     and v3), FictionBook2, and S5, Slidy and Slideous HTML slide shows.
 
-depends_lib-append  \
-                    port:hs-aeson \
-                    port:hs-alex \
-                    port:hs-attoparsec \
-                    port:hs-base64-bytestring \
-                    port:hs-blaze-html \
-                    port:hs-blaze-markup \
-                    port:hs-data-default \
-                    port:hs-extensible-exceptions \
-                    port:hs-happy \
-                    port:hs-highlighting-kate \
-                    port:hs-hslua \
-                    port:hs-http \
-                    port:hs-mtl \
-                    port:hs-network \
-                    port:hs-pandoc-types \
-                    port:hs-parsec \
-                    port:hs-random \
-                    port:hs-syb \
-                    port:hs-sha \
-                    port:hs-tagsoup \
-                    port:hs-temporary \
-                    port:hs-texmath \
-                    port:hs-text \
-                    port:hs-unordered-containers \
-                    port:hs-vector \
-                    port:hs-xml \
-                    port:hs-yaml \
-                    port:hs-zip-archive \
-                    port:hs-zlib
+depends_build-append  \
+                    port:stack
+
+set stack_root      ${workpath}/.stack
+
+post-extract {
+    xinstall -m 0755 -d ${stack_root}
+}
+
+configure.cmd       stack
```

Comment:
I would use full path just in case (`${prefix}/bin/stack`).

Reply:
I didn't want to suggest that this would be the reason for the failure. Just that it's in general a good practice to do it.